airbyte-metrics/reporter/src/main/java/io/airbyte/metrics/reporter/MetricRepository.java

```diff
@@ -140,6 +140,30 @@ having count(*) < 1440 / cast(c.schedule::jsonb->'units' as integer)
         + ctx.fetchOne(queryForAbnormalSyncInMinutesInLastDay).get("cnt", long.class);
   }
 
+  long numberOfJobsRunningUnusuallyLong() {
+    // Definition of unusually long means runtime is more than 2x historic avg run time or 15
+    // minutes more than avg run time, whichever is greater.
+    final var query =
+        """
+        select
```

Comment:
what about trying to do something like https://github.com/airbytehq/airbyte/blob/master/airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java#L640?

Parker started doing this and I found it made SQL queries much more readable. What do you think?

Reply:
yep - added in-sql comments